topics/proteomics/tutorials/metaquantome-data-creation/tutorial.md

```diff
@@ -0,0 +1,644 @@
+---
+layout: tutorial_hands_on
+
+title: metaQuantome Data creation tutorial
+zenodo_link: '(https://doi.org/10.5281/zenodo.4037137)'
+questions:
+ - "How do I perform functional and taxonomy analysis on metaproteomics data?"
+ - "How can I perform quantitation on metaproteomics data?"
+ - "How do I create inputs that can be used in metaquantome to examine differentially expressed proteins?"
+objectives:
+  - "A taxonomy, functional and quantitational analysis of metaproteomic mass spectrometry data."
+time_estimation: "1 hr"
+key_points:
+  - "Use dataset collections"
+  - "With SearchGUI and PeptideShaker you can gain access to multiple search engines"
+  - "Learning the basics of SQL queries can pay off"
+contributors:
+  - subinamehta
+  - timothygriffin
+  - pratikdjagtap 
+  - emmaleith
+  - mariecrane
```

Comment:
please add this contributor to the CONTRIBUTORS.yaml file

Reply:
added marie